### PR TITLE
Move text messages of components to JSON

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

File: src/components/PoolPositionCard/PoolPositionCard.tsx [hunk 1]
```diff
@@ -13,11 +12,11 @@ import { DoubleCurrencyLogo } from 'components';
 import { formatAPY, getAPYWithFee, getOneYearFee } from 'utils';
 import PoolPositionCardDetails from './PoolPositionCardDetails';
 import 'components/styles/PoolPositionCard.scss';
+import { Trans, useTranslation } from 'react-i18next';
 
 const PoolPositionCard: React.FC<{ pair: Pair }> = ({ pair }) => {
+  const { t } = useTranslation();
   const [bulkPairData, setBulkPairData] = useState<any>(null);
-  const { breakpoints } = useTheme();
-  const isMobile = useMediaQuery(breakpoints.down('xs'));
```
Comment (FriskyHamTitz) [inline src/components/PoolPositionCard/PoolPositionCard.tsx:20]: Why did we remove mobile

File: src/components/RemoveLiquidityModal/RemoveLiquidityModal.tsx [hunk 2]
```diff
@@ -222,12 +224,12 @@ const RemoveLiquidityModal: React.FC<RemoveLiquidityModalProps> = ({
...
 
     const liquidityAmount = parsedAmounts[Field.LIQUIDITY];
-    if (!liquidityAmount) throw new Error('missing liquidity amount');
+    if (!liquidityAmount) throw new Error(t('noLiquidity'));
 
     const currencyBIsETH = currency1 === ETHER;
     const oneCurrencyIsETH = currency0 === ETHER || currencyBIsETH;
 
-    if (!tokenA || !tokenB) throw new Error('could not wrap');
+    if (!tokenA || !tokenB) throw new Error('cannotWrap');
```
Comment (FriskyHamTitz) [inline src/components/RemoveLiquidityModal/RemoveLiquidityModal.tsx:232]: I think you meant to use the translation in the error message

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/components/PoolPositionCard/PoolPositionCard.tsx (hunk 1 above)
Comment (totop716) [reply src/components/PoolPositionCard/PoolPositionCard.tsx:20]: Updated css for mobile responsive and removed as this is unnecessary

File: src/components/RemoveLiquidityModal/RemoveLiquidityModal.tsx (hunk 2 above)
Comment (totop716) [reply src/components/RemoveLiquidityModal/RemoveLiquidityModal.tsx]: Fixed this and pushed another commit

**Review by sameepsi — APPROVED** [review]

**Review by sameepsi — APPROVED** [review]

**Review by FriskyHamTitz — APPROVED** [review]

**Pull request merged**